[ERROR] Server:0.12.4-R2

If the raid starts, an error will be reported when reading the map

[ERROR] Server:0.12.4-R2
[ERROR] Trace:
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type undefined
    at Object.openSync (fs.js:432:10)
    at Object.readFileSync (fs.js:342:35)
    at Object.readFileSync (E:\download\Server-development\EmuTarkov-Server.exe:201:55)
    at Object.read (E:\download\Server-development\core\util\json.js:22:16)
    at Object.cost (E:\download\Server-development\src\classes\insurance.js:151:52)
    at Object.getInsuranceCost [as /client/insurance/items/list/cost] (E:\download\Server-development\src\responses\insurance.js:4:74)
    at Router.getResponse (E:\download\Server-development\core\server\router.js:36:44)
    at Server.sendResponse (E:\download\Server-development\core\server\server.js:138:29)
    at Inflate.cb (E:\download\Server-development\core\server\server.js:172:28)
    at Inflate.zlibBufferOnEnd (zlib.js:139:10) {
  code: 'ERR_INVALID_ARG_TYPE'

--- a/src/classes/insurance.js
+++ b/src/classes/insurance.js
@@ -137,6 +137,20 @@ class InsuranceServer {
     }
 }
 
+// TODO: Move to helper functions
+function getItemPrice(_tpl) {
+	let price = 0;
+	if(_tpl in db.templates.items) {
+		let template = json.parse(json.read(db.templates.items[_tpl]));
+		price = template.Price;
+	} else {
+		let item = json.parse(json.read(db.items[_tpl]));
+		price = item._props.CreditsPrice;
+	}
+	
+	return price;
+}
+
 /* calculates insurance cost */
 function cost(info, sessionID) {
     let output = {};
@@ -148,8 +162,7 @@ function cost(info, sessionID) {
         for (let key of info.items) {
             for (let item of pmcData.Inventory.items) {
                 if (item._id === key) {
-                    let template = json.parse(json.read(db.templates.items[item._tpl]));
-                    items[template.Id] = Math.round(template.Price * gameplayConfig.trading.insureMultiplier);
+                    items[item._tpl] = Math.round(getItemPrice(item._tpl) * gameplayConfig.trading.insureMultiplier);
                     break;
                 }
             }
@@ -169,11 +182,9 @@ function insure(pmcData, body, sessionID) {
     for (let key of body.items) {
         for (let item of pmcData.Inventory.items) {
             if (item._id === key) {
-                let template = json.parse(json.read(db.templates.items[item._tpl]));
-
                 itemsToPay.push({
                     "id": item._id,
-                    "count": Math.round(template.Price * gameplayConfig.trading.insureMultiplier)
+                    "count": Math.round(getItemPrice(item._tpl) * gameplayConfig.trading.insureMultiplier)
                 });
                 break;
             }

